src/CMakeLists.txt

```diff
@@ -71,6 +71,19 @@ function(CONSUME_PAL target)
     target_link_libraries(${target} pal)
 endfunction()
 
+# set test_platform linker flags to look for .so files in the current directory 
+# adapted from https://serverfault.com/a/926072
+function(RPATH_ORIGIN target)
+    if (NOT WIN32)
```

Comment:
this would pickup MacOS?  do we have a more specific platform filter?

Reply:
I'm not sure what we'll need for MacOS, but I figure we will cross that bridge when we get there.